docs/source/filter.mdx

```diff
@@ -0,0 +1,33 @@
+# Filter rows in a dataset
+
+Datasets Server provides a `/filter` endpoint for filtering rows in a dataset.
+
+<Tip warning={true}>
+  Currently, only <a href="./parquet">datasets with Parquet exports</a>
+  are supported so Datasets Server can index the contents and run the search without
```

Comment:
this part should be adapted a bit for "filter"

Reply:
I replaced "search" with "filter query". Is it OK?